attalos/imgtxt_algorithms/sampledSM/train_tensorflow.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import numpy as np
+
+from attalos.dataset.dataset import Dataset
+from attalos.dataset.onehot import OneHot
+
+import negsamp
+
+# Constants
+batch_size = 128
+hidden_layer_size = 100
+learning_rate = 0.001
+negative_sample_count = 10
+training_epochs = 10
+
+# Read in the data
+train_image_feature_file = '/local_data/features/image/iaprtc12_train_inception.hdf5'
```

Comment:
We don't want to hardcode paths; is the right answer to build a `main()` that reads from the command line?